### PR TITLE
Updated Array_Rotation 

### PR DESCRIPTION
**Pull request by JasleenRandhawa**
Fixes #

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- I have added a rightRotate() method that rotates the array to its right by d elements

#### Language Used:
- java

#### Files Added:
- None

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

I have updated this code for the issue #1762

### CODE REVIEW

File: Array_Rotation/Array_Rotation.java [hunk 1]
```diff
@@ -21,18 +30,32 @@ private static void Rotate(int arr[], int rotations, int n)
...
 
     public static void main(String args[]) 
     { 
         // Sample input
-        int arr[] = {1, 2, 3, 4, 5}; 
-        int n = arr.length; 
-        Rotate(arr, 2, n);
+        int arr1[] = {1, 2, 3, 4, 5}; 
+        int arr2[] = {1, 2, 3, 4, 5};
+        int n = arr1.length;
```
Comment (MastersAbh) [inline Array_Rotation/Array_Rotation.java]: Please take dynamic inputs

File: Array_Rotation/Array_Rotation.java [hunk 2]
```diff
@@ -21,18 +30,32 @@ private static void Rotate(int arr[], int rotations, int n)
...
-        int arr[] = {1, 2, 3, 4, 5}; 
-        int n = arr.length; 
-        Rotate(arr, 2, n);
+        int arr1[] = {1, 2, 3, 4, 5}; 
+        int arr2[] = {1, 2, 3, 4, 5};
+        int n = arr1.length; 
+        int m = arr2.length;
+        RotateDriverLeft(arr1, 2, n);
+        System.out.println();
+        RotateDriverRight(arr2, 2, m);
```
Comment (MastersAbh) [inline Array_Rotation/Array_Rotation.java]: Rename the functions to RotateArrayLeft & ..Right to be more clear

File: Array_Rotation/Array_Rotation.java (hunk 1 above)
Comment (JasleenRandhawa) [reply Array_Rotation/Array_Rotation.java]: I have formatted the code to take dynamic inputs.

File: Array_Rotation/Array_Rotation.java (hunk 2 above)
Comment (JasleenRandhawa) [reply Array_Rotation/Array_Rotation.java]: I have renamed the functions as per your suggestions

**Comment by JasleenRandhawa** [comment]
@MastersAbh Could you please review the changes?

**Pull request merged**